src/main/java/uk/nhs/cdss/audit/config/AWSConfig.java

```diff
@@ -0,0 +1,20 @@
+package uk.nhs.cdss.audit.config;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.sqs.AmazonSQS;
+import com.amazonaws.services.sqs.AmazonSQSClientBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class AWSConfig {
+
+  @Bean
+  public AmazonSQS sqsClient() {
+    return AmazonSQSClientBuilder
+        .standard()
+        .withRegion(Regions.EU_WEST_2)
```

Comment:
Normally the region wouldn't be hard-coded - it comes from the environment when run in AWS. Locally you can set it in your ~/.aws/config default profile if it's complaining.

Reply:
Yep that's fine, have removed this. Can't test this locally as need dev credentials anyway so few more changes to make